extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -77,6 +78,9 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 	private _refreshButton!: azdata.ButtonComponent;
 	private _databaseTable!: azdata.TableComponent;
 	private _migrationTableSection!: azdata.FlexContainer;
+	// key,value pair of sourceDatabaseName,status
+	private _schemaMigrationStatus: Map<string, SchemaMigrationState> = new Map<string, SchemaMigrationState>();
```

Comment:
think we should move this into sqlSchemaMigrationModel.ts. all schema related info should be encapulated withing the schemaModel